### PR TITLE
wfuzz: add python3-shodan

### PR DESCRIPTION
**Pull request by misuchiru03**

### CODE REVIEW

File: srcpkgs/python3-click-plugins/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'python3-click-plugins'
+pkgname=python3-click-plugins
+version=1.1.1
+revision=1
+archs="noarch"
+wrksrc="click-plugins-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3"
```
Comment (daniel-eys) [inline srcpkgs/python3-click-plugins/template]: Needs `python3-click`.

File: srcpkgs/python3-shodan/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'python3-shodan'
+pkgname=python3-shodan
+version=1.12.1
+revision=1
+archs="noarch"
+wrksrc="shodan-python-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+makedepends="python3-click python3-click-plugins python3-colorama
+ python3-requests python3-XlsxWriter"
```
Comment (daniel-eys) [inline srcpkgs/python3-shodan/template]: These are runtime deps so they need to be in `depends`.

**Comment by daniel-eys** [comment]
Please use one commit per add/change of a package.

**Comment by misuchiru03** [comment]
Cleaned-up templates and added commits.

**Comment by misuchiru03** [comment]
Added `pycompile_module` to each

**Pull request merged**